build.psm1

```diff
@@ -1419,30 +1417,22 @@ function Start-PSxUnit {
             }
         }
 
-        # Run sequential tests first, and then run the tests that can execute in parallel
-        if (Test-Path $SequentialTestResultsFile) {
-            Remove-Item $SequentialTestResultsFile -Force -ErrorAction SilentlyContinue
-        }
-        dotnet test --configuration $Options.configuration --filter FullyQualifiedName~PSTests.Sequential -p:ParallelizeTestCollections=false --test-adapter-path:. "--logger:xunit;LogFilePath=$SequentialTestResultsFile"
-        Publish-TestResults -Path $SequentialTestResultsFile -Type 'XUnit' -Title 'Xunit Sequential'
+        dotnet build --configuration $Options.configuration 
 
-        $extraParams = @()
+        if (Test-Path $ParallelTestResultsFile) {
+            Remove-Item $ParallelTestResultsFile -Force -ErrorAction SilentlyContinue
+        }
 
         # we are having intermittent issues on macOS with these tests failing.
         # VSTS has suggested forcing them to be sequential
```

Comment:
Can we validate if this is still the case. If not, we can removed the condition.

Reply:
Perhap @TravisEz13 can comment this.

Reply:
We would have to try it to know.

Reply:
We could open a tracking issue for the work and don't block the PR.

Also we have many CodeFactor issues. Also we need to have more tests for binary APIs. Also we need to have o\performance tests. I think we should open new tracking issues.

Reply:
Done #8400